test/CMakeLists.txt

```diff
@@ -1,14 +1,21 @@
 add_executable(benchstat
-  bench.cpp)
+        bench.cpp)
 
 target_link_libraries(benchstat
-  PRIVATE
-    Lorina
-)
+        PRIVATE
+        Lorina
+        )
 
 add_executable(ttf-sample ttf_sample.cpp)
 target_link_libraries(ttf-sample
-  PRIVATE
-    SDL2::SDL2
-    SDL2_ttf::SDL2_ttf
-)
+        PRIVATE
+        SDL2::SDL2
+        SDL2_ttf::SDL2_ttf
+        )
+
+
+target_link_libraries(benchstat
+        PRIVATE
+        SDL2::SDL2
+        SDL2_ttf::SDL2_ttf
+        )
```

Comment:
Disable autoindentation of the IDE or at least make it consistent with the working files.